apps/web/package.json

```diff
@@ -24,7 +24,7 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
```

Comment:
Need new react-dom typings?

Reply:
react-dom typings went from 16.9.X to 17.X.  So I dont think so.